components/team_selector_modal/team_selector_modal.tsx

```diff
@@ -177,11 +194,11 @@ export default class TeamSelectorModal extends React.PureComponent {
         );
     }
 
-    renderValue(props) {
+    renderValue(props: { data: any; }) {
         return props.data.display_name;
     }
 
-    renderConfirmModal(show, team) {
+    renderConfirmModal(show: boolean, team: any) {
```

Comment:
Can we update `team` to be be more specific (`Team`) than `any`?

Reply:
done in latest commit